src/pages/edit/ActionsBar.js

```diff
@@ -159,23 +157,13 @@ const EditBar = ({ dashboard, ...props }) => {
     }
 
     const translationDialog = () =>
-        dashboard.id ? (
+        translationDlgIsOpen ? (
             <TranslationDialog
                 className="translation-dialog"
-                d2={d2}
-                open={translationDlgIsOpen}
-                onRequestClose={toggleTranslationDialog}
-                objectToTranslate={{
-                    ...dashboard,
-                    modelDefinition: { name: 'dashboard' },
-                }}
+                onClose={toggleTranslationDialog}
+                objectToTranslate={dashboard}
                 fieldsToTranslate={['name', 'description']}
-                onTranslationError={(err) =>
-                    console.log('translation update error', err)
-                }
                 onTranslationSaved={Function.prototype}
-                insertTheme={true}
-                isOnline={online}
```

Comment:
Does the new dialog support the offline disabling of the Save/Cancel buttons?

Reply:
Good point.
I don't think the current implementation of `TranslationDialog` has offline support, but it won't be difficult to add.
I think the `useOnlineStatus` hook can be used directly in the component in `analytics`.

Reply:
Added offline support for `TranslationDialog` in `analytics`.
Because the `OfflineTooltip` component is needed there, it's been moved from the app to `analytics` and used from there.